app/src/main/java/com/lastaoutdoor/lasta/ui/screen/settings/SettingsComponent.kt

```diff
@@ -31,131 +43,218 @@ fun SettingsComponent(
     updateClimbingLevel: (UserLevel) -> Unit,
     updateHikingLevel: (UserLevel) -> Unit,
     updateBikingLevel: (UserLevel) -> Unit,
+    setUpOrSetting: Boolean // Setup if true, Settings if false
 ) {
-  var selectedLanguage = language
-  var selectedActivity = prefActivity
+  Column(horizontalAlignment = Alignment.CenterHorizontally) {
```

Comment:
Try using a LazyColumn instead! Otherwise, the display couldn't work for all sort of phones, exclusively small ones. Check this result on my phone: 
<img src="https://github.com/LASTA-OUTDOOR/lasta/assets/95919886/700102f6-7d23-4eeb-950a-0785948fa318" width="200">
Furthermore, I don't thing that horizontal centering is a good idea, it would be better to have everything to the left
```suggestion
  LazyColumn(modifier = Modifier.padding(16.dp) {
```

Reply:
The lazycolumn is a very good recommendation, but for the centering, I might have to disagree with that, I think that it would be kind of not coherent with the small settings headers that are already aligned to the left, what I can and will do is put the language in a same centered way as the activity levels so that it seems more cohesive !